src/AirtableRecord.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Yoanbernabeu\AirtableClientBundle;
+
+use DateTime;
+use DateTimeInterface;
+
+class AirtableRecord
+{
+    protected $fields;
+    protected string $id;
+    protected DateTimeInterface $createdTime;
+
+    public function __construct(string $id, $fields, DateTimeInterface $createdTime)
+    {
+        $this->fields = $fields;
+        $this->id = $id;
+        $this->createdTime = $createdTime;
+    }
+
+    public static function fromRecord(array $record): self
+    {
+        return new self($record['id'], $record['fields'], new DateTime($record['createdTime']));
+    }
+
+    public function getFields()
+    {
+        return $this->fields;
+    }
+
+    public function setFields($fields)
```

Comment:
💡 I would remove all setters as object should be created from the API call and should not be updated
My 2 cents ;-)

Reply:
removed all setters excepts setFields, even it is not used anymore ..